.github/workflows/dev-build-deploy.yml

```
@@ -0,0 +1,65 @@
+    environment: dev
+    steps:
+      - name: SSH to server and deploy
+        uses: appleboy/ssh-action@v1.1.0
+        with:
+          host: ${{ secrets.SERVER_HOST }}
+          username: ${{ secrets.SERVER_USERNAME }}
+          key: ${{ secrets.SERVER_SSH_KEY }}
+          port: ${{ secrets.SERVER_PORT }}
+          script: |
+            docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} pull
+            docker-compose -f ${{ secrets.DOCKER_COMPOSE_YAML_PATH }} up -d
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)